frontend/packages/dev-console/src/components/import/ImportSamplePage.tsx

```diff
@@ -29,11 +29,9 @@ import { BaseFormData, GitImportFormData } from './import-types';
 import { detectGitType, validationSchema } from './import-validation-utils';
 import ImportSampleForm from './ImportSampleForm';
 
-type ImportSamplePageProps = RouteComponentProps<{ ns?: string; is?: string; isNs?: string }>;
-
-const ImportSamplePage: React.FC<ImportSamplePageProps> = ({ match }) => {
+const ImportSamplePage: React.FC = () => {
```

Comment:
Observation across the code base:   sometimes we include an empty type object (`React.FC<{}>`) and sometimes no type like here.  We should be consistent.